pallet-logion-loc/src/lib.rs

```diff
@@ -649,28 +647,6 @@ pub mod pallet {
 
 	impl<T: Config> Pallet<T> {
 
-		fn validate_submitter(
-			submitter: &T::AccountId,
-			loc: &LegalOfficerCaseOf<T>
-		) -> DispatchResultWithPostInfo {
-
-			if submitter.eq(&loc.owner) {
-				return Ok(().into());
-			}
-			match &loc.requester {
-				Requester::Account(requester) => {
-					if submitter.eq(&requester) {
-						Ok(().into())
-					} else {
-						Err(Error::<T>::InvalidSubmitter)?
```

Comment:
You may probably also remove `InvalidSubmitter`.

Reply:
I don't think it's a good idea, because enum are actually referenced by their index. In this case,
* We have to regenerate additional types.
* These new additional types are not compatible any more with the old transactions.